.github/workflows/check.yaml

```diff
@@ -0,0 +1,38 @@
+name: Build, linters, and unit tests
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  build-linters-unit-tests:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/setup-python@v2
+      with:
+        python-version: 3.x
+
+    - run: |
+        pip3 install --user --upgrade j2cli
```

Comment:
why is this needed?

Reply:
removed as part of rest of the changes (aligning it to NMPolicy)